extensions/wikia/Recirculation/WikiRecommendations.class.php

```diff
@@ -591,20 +609,21 @@ class WikiRecommendations {
 			'url' => 'http://midnight-texas.wikia.com/wiki/Midnight,_Texas_Wikia',
 		]
 	];
-	
-	public static function getRecommendations( $contentLanguage ) {
-		global $wgDevelEnvironment;
 
+	public static function getRecommendations( $contentLanguage ) {
+		global $wgWikiaEnvironment;
 
-		if ( empty( $wgDevelEnvironment ) ) {
+		if ( $wgWikiaEnvironment === WIKIA_ENV_STAGING ){
+			$recommendations = self::STAGING_RECOMMENDATIONS;
+		} elseif ( $wgWikiaEnvironment === WIKIA_ENV_DEV ) {
+			$recommendations = self::DEV_RECOMMENDATIONS;
+		} else {
 			$recommendations = self::RECOMMENDATIONS['en'];
 
 			if ( array_key_exists( $contentLanguage, self::RECOMMENDATIONS ) ) {
 				$recommendations = self::RECOMMENDATIONS[$contentLanguage];
-			}
+		}
```

Comment:
This bracket should be indented an extra level.

Reply:
Already done in amend to Duplicated dev recommendation fix, I haven't noticed the push was rejected